core/platform/bulk_email/dev_mode_bulk_email_services.py

```diff
@@ -35,8 +35,10 @@ def permanently_delete_user_from_list(user_email: str) -> None:
 
 
 def add_or_update_user_status(
-    user_email: str, can_receive_email_updates: bool,
-    _merge_fields: Dict[str, str], _tag: str
+    user_email: str,
+    _merge_fields: Dict[str, str],
```

Comment:
I don't think I've ever seen _arguments_ being prefixed by underscores. Is this a new convention?

Reply:
Done. It was used for linter, but replaced this with `unused_`